app/src/main/java/com/eis/phoneringer/RingCommands/RingtoneHandler.java

```diff
@@ -0,0 +1,103 @@
+package com.eis.phoneringer.RingCommands;
+
+import android.content.Context;
+import android.media.AudioAttributes;
+import android.media.Ringtone;
+import android.media.RingtoneManager;
+import android.net.Uri;
+import android.os.Build;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+
+/**
+ * Class that interfaces with the RingtoneManager class of the Android library
+ *
+ * @author Alessandra Tonin, Luca Crema, Alberto Ursino
+ */
+public class RingtoneHandler {
+
+    private static Ringtone defaultRingtone;
+    private static RingtoneHandler ringtoneHandler = new RingtoneHandler();
+
+    /**
+     * Private constructor
+     */
+    private RingtoneHandler() {
+    }
+
+    /**
+     * @return the singleton
+     */
+    public static RingtoneHandler getInstance() {
+        return ringtoneHandler;
+    }
+
+    /**
+     * Get the tone of the specified type
+     *
+     * @param context of the application
```

Comment:
Missing `type` param

Reply:
It is wanted because there would be redundant information (because of the description). But I can do the opposite: add the @param type and remove the description

Reply:
I can also leave both...

Reply:
Having @param is preferable to the generic description